ansible/group_vars/all

```diff
@@ -30,7 +30,7 @@ whisk:
 #   runtimes: set of language runtime families grouped by language (e.g., nodejs, python)
 #   blackboxes: list of pre-populated docker action images as "name" with optional "prefix" and "tag"
 #
-runtimesManifest:
+runtimesManifestDefault:
```

Comment:
This change is I think not necessary. We can leave the name as is, the distinction is made via snake_case vs. pascalCase.

Reply:
I'm confuse, I need a new name to hold the default values, take a look below where I define `runtimesManifest`

Reply:
Oh, right. My bad I guess it was a bit early this morning.

Reply:
Can you approve now?